test/functional/eval/timer_spec.lua

```diff
@@ -176,9 +189,9 @@ describe('timers', function()
         let g:val2 += 1
       endfunc
     ]])
-    command("call timer_start(20, 'MyHandler',  {'repeat': 3})")
-    command("call timer_start(40, 'MyHandler2', {'repeat': 2})")
-    run(nil, nil, nil, load_adjust(300))
+    command("call timer_start(2, 'MyHandler',  {'repeat': 3})")
+    command("call timer_start(4, 'MyHandler2', {'repeat': 2})")
+    run(nil, nil, nil, load_adjust(30))
     eq(3,eval("g:val"))
     eq(2,eval("g:val2"))
```

Comment:
@blueyed any reason not to use retry() here? Failed in https://github.com/neovim/neovim/pull/10804 .

Reply:
@justinmk 
Could be done 8around the `eq`s I suppose).
How did it fail?

Reply:
`g:val` did not exist IIRC.